Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md

```diff
@@ -148,10 +148,23 @@ chart to start managing the cluster:
 
 ### **Development Build**
 
-To deploy from a local build from your development environment:
+To deploy from a local build from your development environment you need to follow two steps:
+
+First deploy the operator chart,  in particular to get the CDRs:
```

Comment:
On line 20, the prerequisites mentions that the operator chart must first be installed. Also, there is already a section with these details in the operator chart (see operator-chart.md/#development-build). How about just a simple sentence that links to that other doc?

Reply:
@travisn  wrote:
> On line 20, the prerequisites mentions that the operator chart must first be installed. Also, there is already a section with these details in the operator chart (see operator-chart.md/#development-build). How about just a simple sentence that links to that other doc?


great  points. Initially, I intended to link to the other doc anyway instead of duplicating content anyway,  but I was (and still am)  not quite sure how to link correctly with these generated docs ...

Reply:
done.